queue.go

```diff
@@ -0,0 +1,80 @@
+package cleisthenes
+
+import "fmt"
+
+// Define transaction FIFO queue
```

Comment:
how about
```
type queue interface {}

type memQueue interface {}
```

since maybe HB needs queue mocking because of `RandomPoll`

Reply:
And `randPollQueue` (or other name) only needs to expose two method:

```
type randPollQueue interface {
  Push()
  PollRandom()
}
```

and other methods maybe better change into *private*